Content.Shared/Emag/Components/EmaggedComponent.cs

```diff
@@ -8,4 +9,15 @@ namespace Content.Shared.Emag.Components;
 [RegisterComponent, NetworkedComponent]
 public sealed partial class EmaggedComponent : Component
 {
+    /// <summary>
+    /// Volume of the sound that plays after entity was emagged.
+    /// </summary>
+    [DataField]
+    public float SoundVolume = 20;
+
+    /// <summary>
+    /// Sound that plays after entity was emagged.
+    /// </summary>
+    [DataField]
+    public SoundSpecifier Sound = new SoundCollectionSpecifier("sparks");
```

Comment:
I would keep these on each component being emagged so they can still be customized for each device.